services/ui-src/src/utils/testing/mockRouter.tsx

```diff
@@ -1,7 +1,9 @@
 import React from "react";
 import { BrowserRouter as Router } from "react-router-dom";
 
-export const RouterWrappedComponent: React.FC = ({ children }) => (
+export const RouterWrappedComponent: React.FC<{ children: any }> = ({
+  children,
+}) => (
```

Comment:
React 18 also does away with implicit children. You must declare your children! 😭